src/main/java/homework_1/ConsolePrinter.java

```diff
@@ -0,0 +1,22 @@
+package homework_1;
+
+import ConsoleColors.ConsoleColors;
+
+public class ConsolePrinter {
+    public static void main(String[] args) {
+
+        if (args.length > 0) {
+            for (int i = 0; i < args.length; i++) {
+
+                String toPrint = args[i];
+                if (toPrint.equalsIgnoreCase("error")) {                                //checking for error
```

Comment:
Opt: I think this part is self-explanatory

Reply:
removed